src/code_800AF9B0.c

```diff
@@ -2150,21 +2150,21 @@ void func_800B29D8(void *arg0, u16 arg1) {
                         if (temp_v1_3 != 1) {
                             if (temp_v1_3 != 2) {
                                 if (temp_v1_3 != 3) {
-                                    *phi_t0_2 = 6;
+                                    *phi_t0_2 = CONFIRMATION_SELECTION;
```

Comment:
Potentially `CONFIRMED_OK_SELECTION` as that would relate to what you see in-game. But it's just an idea. Feel free to ignore this comment.

Reply:
Yeah, I like that idea